CHANGELOG.md

```diff
@@ -1077,7 +1077,7 @@ current changes on git with [previous release tags][git_tag_comparison].
 - [Expose mint feature in bevy_math/glam][5857]
 - [Utility methods for Val][6134]
 - [Register missing bevy_text types][6029]
-- [Add additional constructors for `UiRect` to specify values for specific fields][5988]
+- [Add additional constructors for `Frame` to specify values for specific fields][5988]
```

Comment:
Likely didn't mean to edit the changelog.

Reply:
Yep, used a lazy blanket search and replace to fix all the doc comments, that was silly 😅